### ISSUE TITLE
fbs on Pop_OS freezes but won't create release

### ISSUE THREAD

**Issue by travisseymour** [mherrmann/fbs#271]
* Your operating system(s): Pop_OS Linux v21.10
 * Your Python version: 3.9.7
 * Your fbs version: 1.1.0
 * Your PyInstaller version: 4.10
 * Your PyQt / PySide version: Pyside2

I have been rocking and rolling with fbs pro for a while and I love it...sooo many cool projects made much easier to distribute to friends and colleagues. I recently switched from Linux Mint 20.03 to PopOS 21.10. While fbs pro apps run normall, and I'm able to `fb freeze` them just fine, the `fbs installer` and `fbs release` commands complain about my operating system:

> ERROR: Unexpected exit code: 255
> Command line: | venv/bin/fbs installer
> Stderr:       | Your Linux distribution is not supported, sorry. You can run `fbs buildvm` followed by `fbs runvm` to start a Docker VM of a supported distribution.

I understand I could go the vm route as directed, but given that PopOS 21.10 is really just Ubuntu 21.10 with a bunch of tweaks, I'm wondering if there is some easy way to get fbs to just proceed with the release? Is there some config option where I can just state that I'm running Ubuntu?

**Comment by mherrmann** [comment]
What do you get when you type the following into a terminal?

    from fbs_runtime.platform import linux_distribution
    linux_distribution()

I could add it to is_ubuntu() in platform.py to make PopOS be treated as Ubuntu.

**Comment by travisseymour** [comment]
```python
from fbs_runtime.platform import linux_distribution
linux_distribution()
'Pop!_OS'
```

**Comment by travisseymour** [comment]
note that there is a package called distro (pip install distro) that has more info than linux_distribution. It includes a ton of info in it's `info()` as well as a method called `like()` which will return **'ubuntu debian'** on Mint and Pop!_OS (and probably all ubuntu based distros, but I have not personally tested others). Then again, it's another dependency to deal with.

**Comment by travisseymour** [comment]
Note that if I add 'Pop!_OS' to `is_ubuntu()` myself, I am able to successfully create an installer, but not until after I installed the fpm package:

```bash
apt-get install ruby ruby-dev rubygems build-essential
gem install --no-document fpm
```

**Comment by mherrmann** [comment]
Fixed in fbs (and fbs Pro) v1.1.1.
